crates/net/network/src/network.rs

```diff
@@ -148,6 +150,18 @@ impl NetworkHandle {
     }
 }
 
+impl StatusUpdater for NetworkHandle {
+    /// Update the status of the node.
+    fn update_status(&self, height: u64, hash: H256, total_difficulty: U256) {
+        let _ = self.inner.to_manager_tx.send(NetworkHandleMessage::StatusUpdate(StatusUpdate {
+            height,
+            hash,
+            total_difficulty,
+        }));
+    }
```

Comment:
let's also add this as function in impl NetworkHandle so it can be used without the trait

also see `NetworkHandle::send_message`

Reply:
Oops yeah, forgot about that one